theano/gof/destroyhandler.py

```diff
@@ -788,24 +790,27 @@ def fast_destroy(self, app):
         - Allow sequence of view.
         - But don't allow to destroy view
         """
-        return
         dm = getattr(app.op, 'destroy_map', None)
         if not dm:
             return
-        inputs = sum(dm.values())  # list of app's destroyed inputs
+        # inputs = sum(dm.values())  # list of app's destroyed inputs
+        inputs = dm.values()[0]
         for inp_idx in inputs:
             inp = app.inputs[inp_idx]
             if inp.owner:
-                if len(inp.clients() > 1):
-                    raise InconsistencyError()
-                app2 = inp.owner
-                inp_idx2 = app2.outputs.index(inp)
-                d = getattr(app2, 'destroy_map', {}).get(inp_idx2, [])
-                v = getattr(app2, 'view_map', {}).get(inp_idx2, [])
-                dv = d+v
-                assert len(dv) <= 1
-                if len(v) > 0:
-                    raise InconsistencyError()
+                if len(inp.clients) > 1:
+                    self.fail_validate = theano.gof.InconsistencyError(
+                        "Destroyed variable has too many clients")
+                else:
+                    app2 = inp.owner
+                    inp_idx2 = app2.outputs.index(inp)
+                    d = getattr(app2, 'destroy_map', {}).get(inp_idx2, [])
+                    v = getattr(app2, 'view_map', {}).get(inp_idx2, [])
+                    dv = d + v
+                    assert len(dv) <= 1
+                    if len(v) > 0:
```

Comment:
I think there is an error here and it should be "len(dv)" not "len(v)". What do you think?